r_basics_visualize_data/r_basics_visualize_data.md

```diff
@@ -431,7 +431,7 @@ aes(x = a, y = b, color = c)
 
 Let's do a quick check of your understanding of aesthetic mappings.  As a reminder:
 
-* Data visualization visual elements that **don't change** according to the data are **set** , not mapped.  (for example, the color of the background of the graph, which doesn't vary based on the data).
+* Data visualization elements that **don't change** according to the data are **set** , not mapped.  (for example, the color of the background of the graph, which doesn't vary based on the data).
```

Comment:
```suggestion
* Data visualization elements that **don't change** according to the data are **set** are not mapped.  (for example, the color of the background of the graph, which doesn't vary based on the data).
```

Reply:
Hmm I don't understand your suggestion here. The original phrasing makes a lot more sense to me.

Reply:
Although we should get rid of that extra space before the comma...

Reply:
You are totally right, my brain completely elided the "are" in front of **set**

Reply:
```suggestion
* Data visualization elements that **don't change** according to the data are **set**, not mapped.  (for example, the color of the background of the graph, which doesn't vary based on the data).
```